experimental/framework/frs-client/src/FrsClient.ts

```diff
@@ -0,0 +1,163 @@
+        const loader = this.createLoader(containerConfig, containerSchema);
+        const container = await loader.createDetachedContainer({
+            package: "no-dynamic-package",
+            config: {},
+        });
+        await container.attach({ url: containerConfig.id });
+        return this.getFluidContainerAndServices(container);
+    }
+
+    public async getContainer(
+        containerConfig: FrsContainerConfig,
+        containerSchema: ContainerSchema,
+    ): Promise<[FluidContainer, FrsContainerServices]> {
+        const loader = this.createLoader(containerConfig, containerSchema);
+        const container = await loader.resolve({ url: containerConfig.id });
+        if (container.existing === false) {
+            throw new Error("Attempted to load a non-existing container");
+        }
+        return this.getFluidContainerAndServices(container);
+    }
+
+    private async getFluidContainerAndServices(
+        container: Container,
+    ): Promise<[FluidContainer, FrsContainerServices]> {
+        const rootDataObject = await requestFluidObject<RootDataObject>(container, "/");
+        const fluidContainer = new FluidContainer(container, rootDataObject);
+        const containerServices = this.getContainerServices(container);
+        return [fluidContainer, containerServices];
+    }
+
+    private getContainerServices(
+        container: Container,
+    ): FrsContainerServices {
+        return {
+            audience: new FrsAudience(container),
+        };
+    }
+
+    private createLoader(
+        containerConfig: FrsContainerConfig,
+        containerSchema: ContainerSchema,
+    ): Loader {
+        const runtimeFactory = new DOProviderContainerRuntimeFactory(
+            containerSchema,
+        );
+        const module = { fluidExport: runtimeFactory };
+        const codeLoader = { load: async () => module };
+        const urlResolver = new FrsUrlResolver(
+            this.connectionConfig.tenantId,
+            this.connectionConfig.orderer,
+            this.connectionConfig.storage,
+            containerConfig.id,
+            this.tokenProvider,
+        );
+        return new Loader({
+            urlResolver,
+            documentServiceFactory: this.documentServiceFactory,
+            codeLoader,
+            logger: containerConfig.logger,
+        });
+    }
+}
+
+/**
+ * FrsClient static class with singular global instance that lets the developer define
+ * all Container interactions with the Frs service
+ */
+// eslint-disable-next-line @typescript-eslint/no-extraneous-class
+export class FrsClient {
+    private static globalInstance: FrsClientInstance | undefined;
+
+    static init(connectionConfig: FrsConnectionConfig) {
+        if (FrsClient.globalInstance) {
+            debug(
+                `FrsClient has already been initialized. Using existing instance of
+                FrsClient instead.`,
+            );
+            return;
```

Comment:
I think we should throw here.  I don't like the idea that a customer might call `FrsClient.init()` and then be unaware that their subsequent call to `FrsClient.createContainer()` might put their data in an unexpected location because someone else init-ed already.

Reply:
We used to have a `throw` here with an explicit error. However, this was causing issues with React hot reload, as @micahgodbolt, had reported since hot reload would store the in-memory variables since it was "hot" and the app would try to re-initialize the client on reload. We changed this to a log to not block this scenario

Reply:
If the scenario @ChumpChief notes is at all supported, wouldn't that make this global design for the client insufficient?  Do we expect someone will need multiple (different) frs/odsp/whatever connections?

Reply:
I personally don't like the static access (I would prefer to directly create an `FrsClientInstance` for control of this sort of scenario), but I understand that others prefer the static access.  I am not familiar with the hot reload issue but I worry about this being a foot-gun.

Reply:
For this, we export both FrsClient (for the recommended tutorial path) and FrsClientInstance (if you want to create multiple instances and manage them on your own). For the static FrsClient path, @heliocliu you are absolutely right that the multiple connection scenario is not supported by design. In that case, you'd need separate instances.
The idea behind exporting both is that we provide the easy, one connection path but we also expose the underlying multi-instance, multi-connection option if you so choose to use it. For the static FrsClient path, though, the first call to init will be the one that's used to establish the connection, and the other calls will be ignored